pkg/operator/ceph/cluster/crash/crash.go

```diff
@@ -186,3 +198,39 @@ func clusterOwnerRef(clusterName, clusterID string) metav1.OwnerReference {
 		BlockOwnerDeletion: &blockOwner,
 	}
 }
+
+func generateCrashEnvVar(v cephver.CephVersion) corev1.EnvVar {
+	val := fmt.Sprintf("-m $(ROOK_CEPH_MON_HOST) -k %s", keyring.VolumeMount().AdminKeyringFilePath())
+	if v.IsAtLeast(cephver.CephVersion{Major: 14, Minor: 2, Extra: 5}) {
+		val = fmt.Sprintf("-m $(ROOK_CEPH_MON_HOST) -k %s", keyring.VolumeMount().CrashCollectorKeyringFilePath())
+	}
+
+	env := corev1.EnvVar{Name: "CEPH_ARGS", Value: val}
+	return env
+}
+
+func getVolumeMounts(v cephver.CephVersion) corev1.VolumeMount {
+	volumeMounts := keyring.VolumeMount().Admin()
```

Comment:
Can't we skip mounting the admin key if we're not using it? I don't think we should mount it if we're not going to use it.

Reply:
I'm overriding it in the next block if running on 14.2.5, before this release you need the admin key.
Am I missing something obvious here?

Reply:
Oh right, it's overridden. Now I see it!